clients/admin-ui/cypress/fixtures/datasets.json

```diff
@@ -0,0 +1,266 @@
+[
```

Comment:
Did you generate this from demo_resources or something else? Just curious if we should plan on editing these directly or if we're trying to sync them with some other existing examples.

Reply:
Ah yeah `demo_resources` has one dataset, so I copied that a few times to generate multiple. I've normally edited these directly so that it's easier to just generate test data